app/assets/javascripts/tagging.js

```diff
@@ -80,3 +92,19 @@ function initTagForm(deletion_path, selector) {
   return el;
 
 }
+
+function geocodeStringAndPan(string, onComplete) {
+  var url = "https://maps.googleapis.com/maps/api/geocode/json?address=" + string.split(" ").join("+");
+  var Blurred = $.ajax({
+      async: false,
+      url: url
+  });
+  onComplete = onComplete || function onComplete(geometry) {
```

Comment:
Plus, I believe you'll have to assign the onComplete function to run once this Ajax call is complete.  See jQuery docs to see how to attach this so it's run on the response only. Make sense?